### PR TITLE
[RPA-1834] Add feedback page to LP

### PR DESCRIPTION
**Pull request by marcin-depcik**
## Description

image
image
image

Figma:
https://www.figma.com/proto/HRQLYnZ5yBs7nXIuaLuDHr/Strona-z-Chatem?page-id=0%3A1&node-id=17-258&node-type=frame&viewport=1182%2C142%2C0.53&t=noGffeftWgsnL1ek-1&scaling=contain&content-scaling=fixed&starting-point-node-id=17%3A258&show-proto-sidebar=1

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

File: runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/index.ts [hunk 1]
```diff
@@ -41,5 +42,6 @@ const translationResources = {
     ...blogTranslations,
     ...demoTranslations,
     ...usersTranslations,
+    ...feedbackTranslations
```
Comment (bhszewczyk) [inline runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/index.ts]: add coma (as agreed with the team when someone adds next translation it won't affect the previous row

File: runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/index.ts [hunk 2]
```diff
@@ -41,6 +42,7 @@ const translationResources = {
     ...blogTranslations,
     ...demoTranslations,
     ...usersTranslations,
+    ...feedbackTranslations
```
Comment (bhszewczyk) [inline runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/index.ts]: add coma at the end (as above)

File: runbotics/runbotics-orchestrator-ui/src/src-landing/translations/en/feedback/feedback.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "Feedback.Main.HeadSection.WeValueYourOpinion": "We value your opinion",
+    "Feedback.Main.HeadSection.WeWouldBeGrateful": ", so we would be grateful if you could tell us what you think of our participation in the conference.",
```
Comment (bhszewczyk) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/translations/en/feedback/feedback.json]: english suggestion, just to make it feel a little bit less translated xD We value your opinion, so we would appreciate it if you could share your thoughts on our participation in the conference. Please take part in a short survey to help us better meet your needs.

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/FeedbackView/FeedbackView.tsx [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+                        <Typography>
+                            {translate('Feedback.Footer.FindUs')}
+                        </Typography>
+                        <div className={styles.links}>
+                            <Typography className={styles.link}>
+                                <Link
+                                    href="https://www.all-for-one.pl"
+                                    target="_blank"
+                                >
+                                    All For One Poland
```
Comment (bhszewczyk) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/FeedbackView/FeedbackView.tsx]: and here you can use this translation as well (so you can extract this to the companyName variable)

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/FeedbackView/FeedbackView.tsx [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+                                </Link>
+                            </Typography>
+                            <Typography className={styles.link}>
+                                <Link
+                                    href="https://softwarehouse.all-for-one.pl"
+                                    target="_blank"
+                                >
+                                    Enterprise Software House
```
Comment (bhszewczyk) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/FeedbackView/FeedbackView.tsx]: what do you think about moving this to translation as well?

**Review by bhszewczyk — APPROVED** [review]

File: runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/index.ts (hunk 1 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/index.ts]: definitely we want to do that and we want to point out things like that during CRs, but If let's not request changes if that is the only (or a few) thing to fix

**Review by AdriannaSuchomska — APPROVED** [review]

**Review by star-in-the-jar — APPROVED** [review]
Nice job, looking at the screenshots, I think going minimalistic may slightly improve UX - e.g. 
image
could be removed

**Pull request merged**